ramls/request-storage-batch.raml

```diff
@@ -0,0 +1,32 @@
+#%RAML 1.0
+title: Request Storage Batch
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: Batch Request Storage API
+    content: <b>Batch request operations</b>
+
+types:
+  requests-batch: !include requests-batch.json
+  errors: !include raml-util/schemas/errors.schema
+
+traits:
+  validate: !include raml-util/traits/validation.raml
+
+/request-storage-batch:
+  /requests:
+    put:
```

Comment:
The experimental `instance-storage-batch` interface uses `POST` for the batch operation. Why have you chosen `PUT` instead?

Reply:
The `instance-storage-batch` is used to create instances in batch, but this endpoint is going to be used for updates. 

Do you think it should be POST? And which method will we use if we decide to implement endpoint to save requests in batch?

Reply:
Similarly to the conversation about the reordering API, I think Thor API is better suited to POST, because it is really a request to process a batch, rather than a create or replace of a specific resource.

It also keeps it similar to the other experimental batch APIs.

@mreno-EBSCO what do you think?

Reply:
Modified to POST now.